### PR TITLE
fix: Avatar bone rescheduling

### PR DESCRIPTION
**Pull request by dalkia**
## What does this PR change?

Mitigates #779 

This error tends to appear as consequence of other errors. This mitigation stops the spamming of the console, as well as other errors possibly occuring due to said spamming

## How to test the changes?

1. Launch the explorer
2. Get some friends and make them jump to your location

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

**Review by anicalbano — APPROVED** [review]
🟢 First review by QA approved!
Let me know if we need a second one!

image

File: Explorer/Assets/DCL/AvatarRendering/AvatarShape/Components/AvatarTransformMatrixComponent.cs [hunk 1]
```diff
@@ -16,6 +16,9 @@ public struct AvatarTransformMatrixComponent : IDisposable
 
         public void ScheduleBoneMatrixCalculation(Matrix4x4 avatarWorldToLocalMatrix)
         {
+            if (!Handle.IsCompleted)
```
Comment (AlejandroAlvarezMelucciDCL) [inline Explorer/Assets/DCL/AvatarRendering/AvatarShape/Components/AvatarTransformMatrixComponent.cs:19]: I'm out of the loop, but could this mitigation fix hide other errors? The code itself looks good to approve, but I was wondering about that.

**Review by m3taphysics — APPROVED** [review]

File: Explorer/Assets/DCL/AvatarRendering/AvatarShape/Components/AvatarTransformMatrixComponent.cs (hunk 1 above)
Comment (dalkia) [reply Explorer/Assets/DCL/AvatarRendering/AvatarShape/Components/AvatarTransformMatrixComponent.cs:19]: Hi Ale! Its a good point. I dont think its going to hide other errors, but rather complete the job when it has been stopped by a previous error. So, the error should still be there, but this one should not occur as the job will be completed on the next frame

Comment (AlejandroAlvarezMelucciDCL) [reply Explorer/Assets/DCL/AvatarRendering/AvatarShape/Components/AvatarTransformMatrixComponent.cs:19]: That makes sense, thanks for taking the time to add more context, I appreciate it!

**Pull request merged**